pkg/oc/cli/debug/debug.go

```diff
@@ -256,6 +254,7 @@ func (o *DebugOptions) Complete(cmd *cobra.Command, f kcmdutil.Factory, args []s
 
 	o.Builder = f.NewBuilder
 
+	// TODO: not sure what to do with this helper - new duplicate for external versions?
 	o.AddEnv, o.RemoveEnv, err = utilenv.ParseEnv(envArgs, nil)
```

Comment:
@deads2k @soltysh still not sure what to do with this helper, duplicate it to deal with externals? - it depends on other env utils

Reply:
See https://github.com/openshift/origin/pull/20382/files#r204804974